README.md

```diff
@@ -18,6 +18,23 @@ npm install ember-statechart-component
 yarn add ember-statechart-component
 ```
 
+### Optional
+
+To be able to use XState [`state.matches`](https://xstate.js.org/docs/guides/states.html#state-matches-parentstatevalue)
+method in our templates,
+we will first need a `HelperManager` for
+handling vanilla functions.
+[ember-could-get-used-to-this](https://github.com/pzuraq/ember-could-get-used-to-this)
```

Comment:
I don't think we should mention this addon anymore.

There is: https://github.com/NullVoxPopuli/ember-functions-as-helper-polyfill
And in ember 4.3ish, hopefully it won't even be needed due to implementing this RFC: https://github.com/emberjs/rfcs/pull/756

Reply:
Heh, #TIL about ember-function-as-helper-polyfill. I do think however that in some form this needs to stay here, because not everyone's code will be on 4.3+. I will change to use the other addon. Once 4.3(ish) is released, the readme can be changed to give an option to skip this.